packages/labs/virtualizer/src/test/scenarios/element-and-directive-parity.test.ts

```diff
@@ -96,45 +129,67 @@ describe('lit-virtualizer and virtualize directive', () => {
     const ulv: UsingLitVirtualizer = example.querySelector(
       'using-lit-virtualizer'
     )!;
+    const uvd: UsingVirtualizeDirective = example.querySelector(
+      'using-virtualize-directive'
```

Comment:
For my edification, do you know why we need to wait for these elements first with the `until()` call above? The elements are defined in the module, and before the `fixture` call, which itself is awaited. I wouldn't think the `until()`s are needed at all. but if there was a need to wait for an upgrade, `customElements.whenDefined('using-virtualize-directive')` should work too.

I'm asking because the test is so long that any reduction in boilerplate will help new people read it.

Reply:
Yeah this was how I was watching for upgrades from the outside.  I didn't want to select the element until after the upgrade occurred, but I'll see if I can either eliminate the need for those, write a helper, or comment why it's necessary to explicitly await the upgrades for the elements when I do test clean ups in #2788